src/publish/Directory.Build.targets

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project>
+  <Import Project="..\Directory.Build.targets" />
```

Comment:
Why the order change? Are the targets in this file overriding something in the above target files now?

Reply:
no, I just saw that the root Directory.Build.targets is imported at the end here which doesn't make much sense as nested D.B.targets/props file should be incremental to what's already there in the root. But as this is completely unrelated I can undo the change in this PR.

Reply:
The standard pattern I'm aware of is to import the "above" dir.props at the beginning of the file, and the "above" dir.targets at the end of the file, unless you need to override some targets in the "above" dir.targets. This resembles the project file convention and how the default SDK imports work:

```
<Project>
  <Import dir.props>
  [project-specific properties and targets]
  <Import dir.targets>
</Project>
```

I don't have a particular list what kind of problems this prevents (what exactly is more predictable this way)--I don't mind changing, just weird to see it as a change in this PR.

Reply:
I have a different opinion here but we can discuss that at a later point, probably after the consolidation when we do a more overall cleanup.